src/main/java/frc/robot/commands/GoHomeSequence.java

```diff
@@ -4,17 +4,19 @@
 import edu.wpi.first.wpilibj2.command.InstantCommand;
 import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
 import edu.wpi.first.wpilibj2.command.WaitUntilCommand;
+import frc.robot.subsystems.conveyor.Conveyor;
 import frc.robot.subsystems.elevator.Elevator;
 import frc.robot.subsystems.elevator.Elevator.ElevatorPosition;
 import frc.robot.subsystems.intake.Intake;
 import frc.robot.subsystems.intake.Intake.IntakePosition;
+import frc.robot.subsystems.shooter.Shooter;
 
 /**
  * brings the elevator and intake to their respective home positions in a "smart" way so they do not
  * end up attacking each other accidentally
  */
 public class GoHomeSequence extends SequentialCommandGroup {
-  public GoHomeSequence(Intake intake, Elevator elevator) {
+  public GoHomeSequence(Intake intake, Elevator elevator, Conveyor conveyor, Shooter shooter) {
```

Comment:
Did you intend to add these as reqs or no?

Reply:
i thought we needed to command everything ?